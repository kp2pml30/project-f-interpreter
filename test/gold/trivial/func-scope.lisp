(func foo (a)
    (lambda ()
        (setq a (plus a 1))
        a))

(setq f1 (foo 10))
(setq f2 (foo 20))
(f1)
(f1)
(f2)
(f1)
