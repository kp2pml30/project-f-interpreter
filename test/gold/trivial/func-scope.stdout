"int:11"
"int:12"
"int:21"
"int:13"
